src/common/copilot/PowerPagesCopilot.ts

```diff
@@ -335,17 +335,24 @@ export class PowerPagesCopilot implements vscode.WebviewViewProvider {
 
                 this.sendMessageToWebview({ type: 'userName', value: userName });
 
-                let entityName = "";
-                let entityColumns: string[] = [];
+                let metadataInfo = { entityName: '', formName: '' };
+                let entityInfo : string [] = [];
```

Comment:
NIT: Can you remove this added whitespace in order to consistency throughout the code.